.github/linters/codespell.txt

```diff
@@ -339,6 +347,7 @@ opeation
 optin
 orginal
 otherwse
+ources
```

Comment:
I can only see one usage of `ources` which is in `plugins/hypervisors/hyperv/DotNet/ServerResource/WmiWrappers/ROOT.virtualization.v2.Msvm_EthernetSwitchPort.cs`. Better fix in the file and remove `ources` from here.